layouts/community/ortho_4x12/neo2marin/config.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2018-2022 TheAmarin (@theamarin)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Use I2C or Serial, not both */
+#define USE_SERIAL
+//#define USE_I2C
+
+// Tune the tapping term for dual-use `y`/`MOD3` key
+#define TAPPING_TERM 150
+
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif
+
+#ifndef NO_PRINT
+#define NO_PRINT
+#endif
```

Comment:
These are disabled by default, if console is not enabled.  And if console is enabled, you probably want these enabled.
```suggestion
```

Reply:
:+1: